src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
 import './ChatEntry.css';
+// import { useState } from 'react';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+// import Post from './Post.js';
 
 const ChatEntry = (props) => {
+  let flipHeart = () => {
+    if (props.liked) {
+      // props.liked = false;
+      props.updateLikeCallback(-1);
+    } else {
+      // props.liked = true;
+      props.updateLikeCallback(1);
+    }
+  };
```

Comment:
The issue with this is that there is no state keeping track of whether the entry is liked or not. Having a lifted up state in ChatLog or App would allow the props to always be properly up-to-date